components/markets/MarketTimer.tsx

```diff
@@ -1,377 +1,147 @@
-        totalTime: reportingPeriodSec,
-      },
-      OracleReportCooldown: {
-        title: "Oracle has reported",
-        description: "Disputes are open to all",
-        remainingTime: marketStore.reportCooldownDuration,
-        totalTime: disputePeriodSec,
-      },
-      OpenReportWaiting: {
-        title: "Oracle failed to report",
-        description: "Reports open to all",
-        remainingTime: null,
-        totalTime: null,
-      },
-      OpenReportCooldown: {
-        title: "Market reported",
-        description: "Disputes are open to all",
-        remainingTime: marketStore.reportCooldownDuration,
-        totalTime: disputePeriodSec,
-      },
-      Disputed: {
-        title: "Market outcome Disputed",
-        description: "Waiting for authority to report",
-        remainingTime: marketStore.disputeCooldownDuration,
-        totalTime: disputePeriodSec,
-      },
-      AuthorizedReport: {
-        title: "Market outcome reported",
-        description: "",
-        remainingTime: marketStore.disputeCooldownDuration,
-        totalTime: disputePeriodSec,
-      },
-      Resolved: {
-        title: "Market Resolved",
-        description: "Consensus has been reached on the outcome",
-        remainingTime: 0,
-        totalTime: 0,
-      },
-    };
-
-    return (
-      <>
-        {marketStage ? (
-          <ProgressReport
-            title={marketStageCopy[marketStage].title}
-            description={marketStageCopy[marketStage].description}
-            totalTime={marketStageCopy[marketStage].totalTime}
-            remainingTime={marketStageCopy[marketStage].remainingTime}
-            stages={marketStages}
-            currentStageIndex={marketStageIndex}
-          />
-        ) : (
-          <></>
-        )}
-      </>
-    );
+const copy: Record<
+  MarketStage["type"],
+  { title: string; description: string; color: string }
+> = {
+  Proposed: {
+    title: "Market is Proposed",
+    description: "Awaiting approval",
+    color: "bg-yellow-400",
   },
-);
-
-export default dynamic(() => Promise.resolve(MarketTimer), {
-  ssr: false,
-});
+  Trading: {
+    title: "Market is Live",
+    description: "Market is open for trading",
+    color: "bg-green-400",
+  },
+  GracePeriod: {
+    title: "Market Grace Period",
+    description: "Market is cooling down before opening to reports",
+    color: "bg-green-400",
+  },
+  OracleReportingPeriod: {
+    title: "Market ended",
+    description: "Waiting for Oracle report",
+    color: "bg-purple-600",
+  },
+  OpenReportingPeriod: {
+    title: "Oracle has failed to report",
+    description: "Reporting open to all",
+    color: "bg-purple-600",
+  },
+  Disputed: {
+    title: "Market outcome Disputed",
+    description: "Waiting for authority to report",
+    color: "bg-orange-400",
+  },
+  Reported: {
+    title: "Outcome Reported.",
+    description: "The market outcome reported by the oracle",
```

Comment:
I think we should keep something like "Disputes are open to all" so people know they can dispute